src/library/scala/collection/BitSet.scala

```diff
@@ -32,6 +32,7 @@ object BitSet extends BitSetFactory[BitSet] {
   def newBuilder = immutable.BitSet.newBuilder
 
   /** $canBuildFromInfo */
-  implicit def canBuildFrom: CanBuildFrom[BitSet, Int, BitSet] = bitsetCanBuildFrom
+  implicit def canBuildFrom: CanBuildFrom[BitSet, Int, BitSet] = ReusableCBF
+  private[this] val ReusableCBF = bitsetCanBuildFrom
```

Comment:
Could we just use an `implicit val` here? You shouldn't need an extra method unless it's polymorphic.

Reply:
will do

Reply:
done